docs/startup.md

```diff
@@ -0,0 +1,32 @@
+todo: build out config parts (getconfmap).  document how settings and config get rendered
+```mermaid
+flowchart TD
+    A("`**command.NewCommand**`") -->|1| B("`**UpdateSettingsUsingFlags**`")
+    A --> |2| C("`**NewCollector**
+    Creates and returns a new instance of Collector`")
+    A --> |3| D("`**Collector.Run**
+    Starts the collector and waits for its completion.  Also includes the control logic for config reloading and shutdown`")
```

Comment:
I find 'waits for its completion' a bit hard to understand. I would also remove the config reloading bit (I don't find it particularly important to call out when compared to any other part of the Collector's behavior). Maybe just this?
```suggestion
    Starts the collector and blocks until it shuts down`")
```

Reply:
I'm ok cutting it down to this - but I'll add the detail of the control logic to the Collector.Run function comment.